pkg/upgrade/upgrade.go

```diff
@@ -264,7 +264,7 @@ func CreateDefaultDSCI(cli client.Client, platform deploy.Platform, appNamespace
 func UpdateFromLegacyVersion(cli client.Client, platform deploy.Platform, appNS string, montNamespace string) error {
 	// If platform is Managed, remove Kfdefs and create default dsc
 	if platform == deploy.ManagedRhods {
-		fmt.Println("starting deletion of Deloyments in managed cluster")
+		fmt.Println("starting deletion of Deployment in managed cluster")
```

Comment:
Just out of curiosity - is removal of `s` at the end of the word on purpose? Looks like there are couple of items to be deleted. :thinking:

Reply:
it is the spelling of deployment  not about the "s", ref jira issue

Reply:
Well, exactly - that was the reason I asked whether the removal of `s` is also intended here.